timsort.c

```diff
@@ -0,0 +1,30 @@
+#include "lib/timsort_lib.h"
+#include "lib/timsortdata.h"
+#include <cheriintrin.h>
+
+/**
+ * Allocates a large chunk of memory, copies a large chunk of data to it and then sorts using
+ * `timsort`. The example uses two implementations, one that uses explicit parameters to encode run
+ * lengths and another that uses capabilities.
+ * @param arr array to print
+ * @return EXIT_SUCCESS on success. EXIT_FAILURE otherwise
+ */
+int main(int argc, char *argv[])
+{
+
+	int *arr = random_chunk();
```

Comment:
Self-reporting: Forgot to update use of random_chunk(); to take a size parameter. Change incoming.